discovery-provider/alembic/env.py

```diff
@@ -27,6 +27,18 @@
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
 
+kill_running_queries_sql = """
+    BEGIN;
+        SELECT
+            pg_cancel_backend(pid)
```

Comment:
should be a comma here

Reply:
yikes thx -- not sure how this ran successfully w/o it, but it did 😳